src/main/java/me/zeroeightsix/kami/process/AutoObsidianProcess.kt

```diff
@@ -0,0 +1,45 @@
+package me.zeroeightsix.kami.process
+
+import baritone.api.BaritoneAPI
+import baritone.api.IBaritone
+import baritone.api.pathing.goals.GoalNear
+import baritone.api.process.IBaritoneProcess
+import baritone.api.process.PathingCommand
+import baritone.api.process.PathingCommandType
+import me.zeroeightsix.kami.KamiMod
+import me.zeroeightsix.kami.module.modules.misc.AutoObsidian
+
+/**
+ * Created by Xiaro on 13/07/20.
+ */
+class AutoObsidianProcess : IBaritoneProcess {
+
+    private lateinit var baritone: IBaritone
+
+    override fun isTemporary(): Boolean {
+        return true
+    }
+
+    override fun priority(): Double {
+        return 2.0
+    }
+
+    override fun onLostControl() {}
+
+    override fun displayName0(): String {
+        return "AutoObsidian: " + KamiMod.MODULE_MANAGER.getModuleT(AutoObsidian::class.java).state.toString().toLowerCase()
```

Comment:
maybe keep this in the UPPER_CASE (remove .toLowerCase())

Reply:
It looks weird if you have full upper case word with normal words together